portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,17 +15,101 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
+
+  private Comment comment;
+
+  private final static String ENTITY_NAME = "Comments";
+
+  private final static String NAME_PROPERTY = "name";
+  private final static String COMMENT_PROPERTY = "comment";
+  private final static String TIMESTAMP_PROPERTY = "timestamp";
+  private final static String UPVOTES_PROPERTY = "upvotes";
+  private final static String DOWNVOTES_PROPERTY = "downvotes";
+
+  private final static String ID_PROPERTY = "id";
+
+
+  @Override
+  protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    response.setContentType("application/json");
+    
+    
+    Entity commentEntity = new Entity(ENTITY_NAME);
+
+    String name = request.getParameter("name");
+    String commentText = request.getParameter("comment");
+    long timestamp = System.currentTimeMillis();
+    commentEntity.setProperty(NAME_PROPERTY, name);
+    commentEntity.setProperty(COMMENT_PROPERTY, commentText);
+    commentEntity.setProperty(TIMESTAMP_PROPERTY, timestamp);
+    commentEntity.setProperty(UPVOTES_PROPERTY, 0);
+    commentEntity.setProperty(DOWNVOTES_PROPERTY, 0);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Does the `datastore` need to be re-initialized for every request?

Reply:
No, fixing it now